### PR TITLE
Fix/cleanup mappings and settings2

### PR DESCRIPTION
**Pull request by jesperstarkar**
First stage ready to be merged.

**Should be squashed on merge**

**For review** 
- CircleCI complains
- Read through changes in logic (layers and cofig basically)

**In next round (will open another PR)**
- Better definition of Sisyfos layers
- DVE setup etc
- Source- and outputlayers

### CODE REVIEW

**Comment by jesperstarkar** [comment]
Comment updated, ready for a review and squash/merge

**Comment by thomasslee97** [comment]
I'm happy with this

File: package.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
     "webpack-cli": "^3.1.2"
   },
   "dependencies": {
+    "handlebars": "^4.3.0",
```
Comment (Julusian) [inline package.json]: why is handlebars needed?
Also its in resolutions below, doesn't a yarn upgrade fix any audit issues? (its not been a problem in anything else)

File: src/tv2_afvd_showstyle/getRundown.ts [hunk 2]
```diff
@@ -384,14 +414,44 @@ function getBaseline(config: BlueprintConfig): TSRTimelineObjBase[] {
 					},
 					properties: {
 						tie: false,
-						preMultiply: true,
+						preMultiply: false, // @ todo: set up the proper parameters for a good key here
```
Comment (Julusian) [inline src/tv2_afvd_showstyle/getRundown.ts]: why isnt this premultiplied?

**Pull request merged**